tests/rgw/sanity_rgw_multisite.py

```diff
@@ -31,7 +30,7 @@ def run(**kw):
 
     test_folder = "rgw-ms-tests"
     test_folder_path = f"/home/cephuser/{test_folder}"
-
+    home_dir_path = f"/home/cephuser/"
```

Comment:
Could you remove the `f` format string in this line as there are variables to format in the string?